docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,106 @@
-use the ``sshd_config`` resource from `augeasproviders_ssh`_.  This is what the
-SIMP ``ssh`` module uses internally to manage the ``/etc/ssh/sshd_config``
-file, and you can use it to set any options ``ssh::server::conf`` does not
-manage.
-
-For instance, to set the sshd ``LogLevel`` option to ``VERBOSE``:
+Starting with version **6.4.0** of the ``simp-ssh`` module, you can use the
+`sshd_config`_ resource from the ``augeasproviders_ssh`` module to manage
+settings that the module does specifically define. To manage global sshd_config
+settings that are not included in the ``ssh::server::conf`` class, specify them
+in Hiera with the ``ssh::server::conf::custom_entries`` parameter as follows:
 
-.. code-block:: puppet
-
-   # VERBOSE will log SSH key fingerprints used for logins
-   sshd_config { 'LogLevel' : value => 'VERBOSE' }
-
-
-Mixing ``ssh::server::conf`` and ``sshd_config``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Some SSH server configurations may require a combination of
-``ssh::server::conf`` (for options that SIMP manages) and ``sshd_config``
-resources (for additional options). The following example configures the
-``/etc/ssh/sshd_config`` keys ``GSSAPIAuthentication``, ``GSSAPIKeyExchange``,
-and ``GSSAPICleanupCredentials`` with a value of "**yes**":
+.. NOTE::
 
-In Hiera:
+   Due to their complexity, ``Match`` entries are not supported and will
+   need to be added using ``sshd_config_match`` resources as described in
+   ``augeasproviders_ssh`` module.
 
 .. code-block:: yaml
 
-   # GSSAPIKeyExchange + GSSAPICleanupCredentials are managed via sshd_config
-   ssh::server::conf::gssapiauthentication: true
-
-In Puppet:
-
-.. code-block:: puppet
+  ssh::server::conf::custom_entries:
+    AllowAgentForwarding: "yes"
+    AuthorizedPrincipalsCommand: "/usr/local/bin/my_command"
+ 
+.. NOTE::
 
-   include 'ssh::server'
-
-   sshd_config {
-    default:
-      ensure => 'present',
-      value  => 'yes',
-    ;
-    # GSSAPIAuthentication is managed via `ssh::server::conf::gssapiauthentication`
-    ['GSSAPIKeyExchange', 'GSSAPICleanupCredentials']:
-      # use defaults
-    ;
-   }
+   This parameter is **not validated**. Be careful to only specify settings
+   that are only allowed for your particular SSH daemon and avoid duplicate
+   declaration of resources already specified. Invalid options may cause the
+   ssh service to fail on restart.
 
+There are also number of SIMP specific parameters, such as whether the system
+is FIPS enabled, has a firewall, or utilizes LDAP. The ``ssh::server::config``
+class utilizes ``simplib::lookup`` as well as retrieving parameters from the
+``ssh::server::params`` class to help integrate the ``simp-ssh`` module within
+the larger SIMP environment. This includes things like determining appropriate
+fallback ciphers for inclusion in ``sshd_config``, ensuring proper
+authentication, and that SSH traffic passes through the firewall. More
```

Comment:
Multiple items or clauses in a series should be parallel (have the same form).  This is not parallel:
* determining (noun phrase)
* ensuring (noun phrase)
* and that (declarative clause)

Please replace "`and that SSH traffic passes through the firewall` with `and allowing SSH traffic through the firewall`.